Distributions/OpacitySource/wlOpacityInterpolationModule.f90

```diff
@@ -3,7 +3,9 @@ MODULE wlOpacityInterpolationModule
   USE wlKindModule, ONLY: &
     dp
   USE wlInterpolationModule, ONLY: &
-    LogInterpolateOpacity_2D1D2D
+    LogInterpolateOpacity_2D1D2D,
```

Comment:
I think we should either:
a) move the `LogInterpolateOpacity_*` routines into this module
or
b) rename the `LogInterpolateOpacity_*` routines to fit the naming conventions used by the other routines in `wlInterpolationModule`.

I think I prefer option (b), but could be convinced otherwise.
Thoughts @endeve ?

Reply:
I had put an wlInterpolateOpacity_Brem routine in that module, but never actually use it anywhere (wlInterpolateOpacity_NES and wlInterpolateOpacity_Pair are not used anywhere either).. This whole module might be removed actually...

Reply:
@endeve @ranchu1 do you agree that wlOpacityInterpolationModule.f90 can be removed altogether?

Reply:
I'm not aware of any place call these two routines (wlInterpolateOpacity_NES and wlInterpolateOpacity_Pair). Correct me if I'm wrong.

Reply:
Yes, a quick grep doesn't show them being used anywhere...